examples/demo/src/authProvider.ts

```diff
@@ -13,7 +13,7 @@ const authProvider: AuthProvider = {
     checkError: () => Promise.resolve(),
     checkAuth: () =>
         localStorage.getItem('username') ? Promise.resolve() : Promise.reject(),
-    getPermissions: () => Promise.resolve(),
+    getPermissions: () => Promise.resolve([]),
```

Comment:
Why do we need it? getPermissions should be able to return undefined

Reply:
Then we must handle the undefined case in the hook

Reply:
Then you need to revert this change